### PR TITLE
Bug 1192186 - Fix confirm dialog when recording via an activity.

### PR DESCRIPTION
**Pull request by aosmond**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a6db01f9d2cf7839463d5706f50f10b142289b76

File: apps/camera/js/controllers/confirm.js [hunk 1]
```diff
@@ -81,6 +81,18 @@ ConfirmController.prototype.bindEvents = function() {
   this.app.on('newmedia', this.onNewMedia);
 };
 
+ConfirmController.prototype.onRecordingChange = function(recording) {
+  if (!this.activity.pick) { return; }
+
+  if (recording === 'starting') {
+    this.recorded = true;
```
Comment (justindarc) [inline apps/camera/js/controllers/confirm.js]: For clarity, we should probably rename `this.recorded` to `this.recording`.

File: apps/camera/js/controllers/camera.js [hunk 2]
```diff
@@ -97,6 +97,18 @@ CameraController.prototype.bindEvents = function() {
...
+CameraController.prototype.onRecordingChange = function(recording) {
+  var active;
+  if (recording === 'started') {
+    active = true;
+  } else if (recording === 'stopped') {
+    active = false;
+  } else {
+    return;
+  }
+  this.app.set('recording', active);
```
Comment (justindarc) [inline apps/camera/js/controllers/camera.js]: Shouldn't we just set the `app` 'recording' property to the actual recording state instead of a boolean?

Comment (justindarc) [inline apps/camera/js/controllers/camera.js]: Disregard. There's a lot of CSS that depends on this being a boolean.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dacb56f2b4c832c09d1c5b4cd9a5f2ee1e9c34bf

**Pull request merged**